core/src/main/java/nl/nn/adapterframework/webcontrol/api/SendJmsMessage.java

```diff
@@ -212,4 +192,51 @@ private void processMessage(JmsSender qms, String messageId, String message) thr
 			throw new ApiException("Error occured on closing connection", e);
 		}
 	}
+	
+	
+	private Optional<String> getAttributeValue(String name, List<Attachment> attachmentList) {
+		return this.getFile(name, attachmentList).map(v -> {
+			return Optional.ofNullable(v.toString());
+		}).orElse(Optional.empty());
+	}
+	
+	private Optional<InputStream> getFile(String name, List<Attachment> attachmentList) {
```

Comment:
I would expect an inputstream or null, there's nothing optional about that?

Reply:
it is an inputStream wrapped in Optional. it meas null or a inputStream class